src/main/webapp/WEB-INF/view/addUserBox.jsp

```diff
@@ -21,7 +21,7 @@
       </div>
       <div class="modal-footer">
         <button id="AddUsersModalCloseButton" type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
```

Comment:
We'll need to change references to "addUsers" or "add-users" everywhere. Maybe "setUsers"?